modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -72,9 +72,7 @@ rc = lcb_open(instance, name, strlen(name)),
 
 If you installed the travel sample data bucket, substitute _travel-sample_ for _bucket-name_.
 
-The 3.1 SDK is ready for the planned introduction of xref:concept-docs:collections.adoc[Collections] in Couchbase Server 7.0 -- the xref:7.0@server:introduction:whats-new.adoc#whats-new-server-700[beta release of this is available now].
-The latest release, 6.6, brings a limited _Developer Preview_ of Collections, allowing Documents to be grouped by purpose or theme, according to specified _Scope_.
-Here we will use `NULL` for default collection, which covers the whole Bucket.
+The 3.1 SDK supports full integration with the xref:concept-docs:collections.adoc[Collections] feature in the latest release of the Couchbase Data Platform, Couchbase Server 7.0. This includes complete support of Collections, allowing Documents to be grouped by purpose or theme, according to a specified _Scope_. Here we will use the `users` collection within the `tenant_agent_00` scope from `travel-sample` bucket as an example.
```

Comment:
Is this what we call it now?  @RichardSmedley or other docs folks can editorialize as needed.

Reply:
No, we've gone back to Server - Data Platform just shows how long since this page was looked at :'(